server/httpin/dialog/install.go

```diff
@@ -176,7 +176,7 @@ func (d *dialog) handleInstall(w http.ResponseWriter, req *http.Request) {
 
 	_, out, err := d.proxy.InstallApp(client, sessionID, cc, noUserConsentForOAuth2, secret, "")
 	if err != nil {
-		d.mm.Log.Warn("Failed to install app", "app_id", cc.AppID, "error", err.Error())
+		d.log.WithError(err).Warnw("Failed to install app", "app_id")
```

Comment:
Are you missing here `cc.AppID`? Or is it being passed somehow in a different way?

Reply:
TY for the catch